docs/change.md

```diff
@@ -0,0 +1,168 @@
+# Change Management and Deprecation Cycles
+
+> ποταμοῖσι τοῖσιν αὐτοῖσιν ἐμβαίνουσιν ἕτερα καὶ ἕτερα ὕδατα ἐπιρρεῖ
+> (Upon those who step into the same stream ever different waters flow)
+>
+> ~ Heraclitus
+
+!!! info
+
+    **TL;DR: Change happens.**
+
+    Sometimes such change is a breaking change. Sometimes such change means
+    retiring capabilities.
+
+    **This document explains how we're going to manage change carefully,
+    collaboratively and compassionately.**
+
+All code changes.
+
+To ensure PyScript changes are smooth, dependable and well advertised within
+our community of users, this document outlines our approach in two areas:
+
+1. Changes and additions to our codebase.
+2. Deprecation of existing features as they are replaced or become redundant.
+
+Ultimately, and out of necessity as a web-based project, our aim is for a
+small, neat, and comprehensible code base, created through engagement with our
+community.
+
+Each release of PyScript is given a [calver](https://calver.org/) release
+number so folks using PyScript can immediately see how up-to-date their
+referenced PyScript is. For example, release number `2024-10-2` is the second
+release made in October, 2024.
+
+Furthermore, **calver allows users to pin the version of PyScript used in their
+project to a specific version**. Since a release never changes once released,
+users can be confident PyScript won't shift from under them.
+
+**We recommend developers using PyScript keep the version of PyScript they are
+using up to date**. To ensure this process is smooth, this document was
+created to describe what you can expect from us.
```

Comment:
We know that this is not necessarily enough though ... pinned dependencies are the key to make both this statement and the desired outcome future proof.

Reply:
+100 here. We should advise pinning the PyScript version AND dependencies as a first step and then the PyScript version can float as suggested